.flowconfig

```diff
@@ -15,7 +15,6 @@ module.name_mapper='^@utils\/\(.*\)$' -> '<PROJECT_ROOT>/src/utils/\1'
 module.name_mapper='^@constants\/\(.*\)$' -> '<PROJECT_ROOT>/src/constants/\1'
 module.name_mapper='^@commands\/\(.*\)$' -> '<PROJECT_ROOT>/src/commands/\1'
 exact_by_default=true
-module.file_ext=.ts
```

Comment:
Why did you remove this?

Reply:
add ts extenxion break `go to definition`  and `coverage` of flow vscode plugin